work12/index.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+const wrap = document.getElementById("js-wrap")
+const ul = document.getElementById("js-list");
+const button = document.getElementById("js-button");
+// const url = "https://myjson.dit.upm.es/api/bins/ほげほげajy3";
+// const url = "https://myjson.dit.upm.es/api/bins/bu5z";
+const url = "https://myjson.dit.upm.es/api/bins/2hj3";
+
+async function getData() {
+  try {
+    const response = await fetch(url);
+    if (response.ok) {
+      const json = await response.json();
+      return json;
+    } else {
+      throw new Error(response.statusText);
+    }
+  } catch (e) {
+    throw new Error(e);
+  }
+}
+
+async function getListData() {
+  loading();
+  let listData;
+  try {
+    listData = await getData();
+  } catch (e) {
+    wrap.textContent = `エラー内容:${e.message}`;
+  } finally {
+    hideLoading();
+  }
+  if (listData.data.length === 0) {
+    wrap.textContent = "data is empty";
+    return;
+  }
+  createElement(listData);
+}
+
+function hideLoading() {
+  ul.style.backgroundImage = "none";
+}
+
```

Comment:
It would be better to remove style `height: 100px` specified in the loading image as well.

Reply:
@kanakonagiri 
So that's it
If only backgroundImage = "none", the height remains.
It was a blind spot.

commit: 768c183

Reply:
Instead of placing an empty element here in html, I think you can insert it at the position specified by `js` (for example, before or after `js-wrap`).
By doing so, I think it will be better without extra elements.